packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -1290,4 +1293,9 @@ export class AttributesController extends ConsumptionBaseController {
 
         return ownSharedAttributeSuccessors;
     }
+
+    public async getAttributeTagCollection(): Promise<AttributeTagCollection> {
+        const backboneAttributeTagCollection = (await this.attributeTagClient.getBackboneAttributeTagCollection()).value;
+        return AttributeTagCollection.fromAny(backboneAttributeTagCollection);
```

Comment:
You can add a from-method to AttributeTagCollection and use that. I think this is how we usually do it.